deviceModeInit.js

```diff
@@ -187,15 +186,16 @@ var rudderTracking = (function () {
       "json"
     );
   }
-  function checkCookieNeedsToBeUpdated() {
-    const oneHourTimeInMilliSeconds = 60 * 60 * 1000;
+  function getTimeForCookieUpdate() {
+    const cookieUpdateFixedInterval = 50 * 60 * 1000; // 50 mins
     const currentTime = Date.now();
     const prev_rs_shopify_cart_identified_at = cookie_action({
       action: "get",
       name: "rs_shopify_cart_identified_at",
     });
-    return currentTime - prev_rs_shopify_cart_identified_at > oneHourTimeInMilliSeconds;
+    return cookieUpdateFixedInterval + prev_rs_shopify_cart_identified_at  - currentTime;
   }
```

Comment:
Please explain what does this function trying to do

Reply:
This will return the time after which rudderIdentifier will get fired.